<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shopCarNumber</class>
 <widget class="QWidget" name="shopCarNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="add_bt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus_bt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_bt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_bt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="failed_lb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>115</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>操作失败</string>
   </property>
  </widget>
  <widget class="QLabel" name="sailNumber">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>99999</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="need_line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>99999</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
